mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.DeleteOrder = (req, res) => {
+  Order.remove({ _id: req.params.orderId }) 
+    .exec((error, result) => {
+      if (error) {
+        return res.status(200).json({ 
```

Comment:
The status code should be 400 for error responses, not 200. It should be `return res.status(400).json({ ... });`